### PR TITLE
WalletLedgerInitComponent styling

### PR DESCRIPTION
**Pull request by banciur**
I'm creating this early PR as I changed base font size that might affect many dimensions so we can work on same base.

Next one will be ledger address chooser

screencapture-localhost-9090-login-ledger-1518547413052
screencapture-localhost-9090-login-ledger-1518547444055

### CODE REVIEW

File: typings/modules.d.ts [hunk 1]
```diff
@@ -8,3 +8,7 @@ declare module "web3-provider-engine/subproviders/hooked-wallet";
 
 declare module "ledger-wallet-provider";
 declare module "eth-lightwallet";
+
+declare module "*.png";
```
Comment (Moejoe90) [inline typings/modules.d.ts]: Is this the best way to declare extentions

Comment (krzkaczor) [reply typings/modules.d.ts]: Yes, this is just to trick TSC into compiling the code but it should be moved to file called `files.d.ts`. Where actually png is already defined...

File: app/components/walletSelector/WalletLedgerInitComponent.tsx [hunk 2]
```diff
@@ -1,54 +1,75 @@
...
+      <Col>
+        <h1 className="text-center">Logging in with Nano Ledger</h1>
+      </Col>
+    </Row>
     {errorMessage && (
-      <Alert color="info">
-        <h4>Connection status:</h4>
-        <p data-test-id="ledger-wallet-error-msg">{errorMessage}</p>
-      </Alert>
+      <WarningAlert className="my-4">
```
Comment (krzkaczor) [inline app/components/walletSelector/WalletLedgerInitComponent.tsx]: this should be centered

Comment (krzkaczor) [inline app/components/walletSelector/WalletLedgerInitComponent.tsx]: what does `my-4` mean? is it typo?

File: app/components/walletSelector/WalletLedgerInitComponent.tsx [hunk 3]
```diff
@@ -1,54 +1,75 @@
...
-        </li>
-      </ul>
-    </div>
+  <>
+    <Row>
+      <Col>
+        <h1 className="text-center">Logging in with Nano Ledger</h1>
+      </Col>
+    </Row>
     {errorMessage && (
```
Comment (krzkaczor) [inline app/components/walletSelector/WalletLedgerInitComponent.tsx]: in separate PR i will get rid of this and always show some error message

File: typings/modules.d.ts (hunk 1 above)
Comment (banciur) [reply typings/modules.d.ts]: This is webpack magic. Then it treats file as module and you can import it in code.

File: app/components/walletSelector/WalletLedgerInitComponent.tsx [hunk 4]
```diff
@@ -1,54 +1,75 @@
...
+        desc="Use left and right key to enter numbers and press 2 keys at the same time to confirm the code"
+      />
+      <InitStep
+        header="3. Pick Ethereum"
+        img={imgStep3}
+        desc="Click on arrows to scroll  apps and pick Ethereum icon. Press 2 keys at the same time to confirm"
+      />
+    </Row>
+    <Row className="mt-5">
+      <Col className={cn("text-center text-md-right", styles.contact)}>
```
Comment (krzkaczor) [inline app/components/walletSelector/WalletLedgerInitComponent.tsx]: this should be close to the bottom border. Maybe you need to change padding settings

File: app/components/walletSelector/WalletLedgerInitComponent.tsx (hunk 2 above)
Comment (banciur) [reply app/components/walletSelector/WalletLedgerInitComponent.tsx]: It's spacing class from (bootstrap)[https://getbootstrap.com/docs/4.0/utilities/spacing/] `y - for classes that set both *-top and *-bottom`

File: app/components/walletSelector/WalletLedgerInitComponent.tsx (hunk 4 above)
Comment (banciur) [reply app/components/walletSelector/WalletLedgerInitComponent.tsx]: fixed

File: app/components/walletSelector/WalletLedgerInitComponent.tsx (hunk 2 above)
Comment (banciur) [reply app/components/walletSelector/WalletLedgerInitComponent.tsx]: fixed

File: typings/modules.d.ts (hunk 1 above)
Comment (banciur) [reply typings/modules.d.ts]: I removed those

File: app/components/walletSelector/WalletLedgerInitComponent.tsx (hunk 2 above)
Comment (krzkaczor) [reply app/components/walletSelector/WalletLedgerInitComponent.tsx]: this is neat i didn't know about this :D

File: app/styles/variables.scss [hunk 5]
```diff
@@ -15,6 +15,11 @@ $smaller-font-size: .8125rem;
 
 $shadow:  0 .125rem .25rem 0 rgba(0, 0, 0, .1);
 
+// Bootstrap variable overrides
+$body-color: $denim;
+$body-bg: $pale-grey-50;
+$font-family-base: "Open Sans", sans-serif;
+$font-size-base: .9375rem !default;
```
Comment (krzkaczor) [inline app/styles/variables.scss]: !default here is not needed

Comment (banciur) [reply app/styles/variables.scss]: This is how bootstrap does it in their source code. - https://github.com/twbs/bootstrap/blob/v4-dev/scss/_variables.scss#L235

**Pull request merged**